src/guide/troubleshooting/configuration-issues.md

```diff
@@ -96,3 +96,23 @@ account_ either, unless you submit an instruction.
 These sorts of subtle mistakes can be avoided, for example, by
 deserialising directly from string literals, or by generating a fresh
 key-pair in places where it makes sense.
+
+## SCALE codec errors
+
+You may encounter SCALE codec errors when there's a discrepancy
+between the data structure on the client and the peer sides.
```

Comment:
I would rephrase. SCALE codec errors appear when the versions of the data structures differ between the client and the peer sides.